### DISCUSSION TITLE
Pan setting presets

### DISCUSSION THREAD

**Discussion by rickrowan** [jamulussoftware/jamulus#2152] [Ideas]
Especially for choral work, setting individual users to different L/R balance positions makes a huge difference in ones ability to hear the other parts.  The experience is akin to standing spatially in the group.  With the new capability to host many users, the task of setting individual balance levels can be daunting.

Suggstion:  Having a preset or auto set pan-all button such as:  a) random setting for each client or b) set each client to vary from Left to Right as they appear from left to right across the client window.  This is probably best, especially if clients are sorted by part.

**Comment by passing** [comment]
To solve the issue described, I could also imagine an alternative interface where you would not have musicians in columns with control sliders, but instead have a rectangular space that corresponds to the pan setting left-to-right and the volume top-to-bottom. Then the clients would be dots/circles on that space that you could move around individually or in groups to change their volume and balance positions.
Could be a quite intuitive way to control the different channels by just dragging them around...

**Comment by menzels** [comment]
I am working on an implementation suggestion for this suggestion:

> Suggstion: Having a preset or auto set pan-all button such as: a) random setting for each client or b) set each client to vary from Left to Right as they appear from left to right across the client window. This is probably best, especially if clients are sorted by part.

**Comment by menzels** [comment]
Working on this feature, i found that i like to keep my own signal in the middle. 
What do you think? is that a good default or do we need to make that configurable?

**Comment by rickrowan** [comment]
I usually find I move myself a bit off-centre but not a lot.  Typically, I don't want anyharmonic sound centred.  I find they need to be just a little off centre do remain nicely distinguishable.  That said, it is not hard to adjust just yourself either way.  I's go with your idea and leave yourself alone.

**Comment by gilgongo** [comment]
Migrating to discussion until such time as a desired spec/PR can be arrived at.
